content_cafe.py

```diff
@@ -45,27 +49,31 @@ class ContentCafeCoverageProvider(BibliographicCoverageProvider):
     INPUT_IDENTIFIER_TYPES = [Identifier.ISBN]
     DATA_SOURCE_NAME = DataSource.CONTENT_CAFE
     
-    def __init__(self, collection, api=None, mirror=None, **kwargs):
+    def __init__(self, collection, api=None, replacement_policy=None, **kwargs):
         """Constructor.
 
         :param collection: A Collection.
         :param api: A ContentCafeAPI.
-        :param mirror: A MirrorUploader.
+        :param replacement_policy: A ReplacementPolicy.
         :param kwargs: Any extra arguments to be passed into the
             BibliographicCoverageProvider superconstructor.
         """
+        _db = Session.object_session(collection)
+        if not replacement_policy:
+            mirror = MirrorUploader.sitewide(_db)
+            replacement_policy = ReplacementPolicy.from_metadata_source(
+                mirror=mirror
+            )
+
         # We pass in registered_only=True because we don't need to cover
         # every single ISBN in the system (most of which are alternate
         # ISBNs found in OCLC Linked Data), only the ISBNs that a
         # client specifically asked to look up.
         super(ContentCafeCoverageProvider, self).__init__(
-            collection=collection, registered_only=True, **kwargs
+            collection=collection, registered_only=True,
```

Comment:
I think I can remove this `registered_only` thing -- alternate ISBNs found in OCLC Linked Data won't be cataloged in any collection.